test/units/modules/remote_management/oneview/test_oneview_storage_system.py

```diff
@@ -0,0 +1,292 @@
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+import yaml
+from ansible.compat.tests import unittest, mock
```

Comment:
Suggestion: add a blank line here to separate `import X` and `from X import` types of import